hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/IntegrationTestBase.kt

```diff
@@ -247,11 +247,11 @@ data class PrisonerBuilder(
   val currentIncentive: CurrentIncentive? = null,
 ) {
 
-  private fun getOffenderBookingTemplate(): OffenderBooking =
-    GsonConfig().gson().fromJson("/templates/booking.json".readResourceAsText(), OffenderBooking::class.java)
+  private fun getOffenderBookingOldTemplate(): OffenderBookingOld =
+    GsonConfig().gson().fromJson("/templates/booking.json".readResourceAsText(), OffenderBookingOld::class.java)
 
-  fun toOffenderBooking(): String = GsonConfig().gson().toJson(
-    getOffenderBookingTemplate().copy(
+  fun toOffenderBookingOld(): String = GsonConfig().gson().toJson(
+    getOffenderBookingOldTemplate().copy(
       offenderNo = this.prisonerNumber,
```

Comment:
Should previously have done this in previous PR when we renamed the old model to OffenderBookingOld - jsut so happens that none of the tests include properties that differ between the old and new models.

This will eventually move to stubbing the new OffenderBooking model once we have switched to the new endpoint.